app/views/spectrums/new.html.erb

```diff
@@ -126,6 +126,7 @@
         <input name="lon" type="hidden" id="lon" />
 
         <a href="#" data-loading-text="Saving..." id="uploadBtn" class="btn btn-primary btn-large" onClick="$W.save(true)">Upload with selected calibration</a>
+        <br><br>
```

Comment:
Hi @drepram, using the `br` tag to add spacing is not recommended. We generally use CSS to solve this.